localstack/services/s3/s3_listener.py

```diff
@@ -380,12 +389,58 @@ def send_notification_for_subscriber(
             )
         except Exception:
             LOGGER.warning(
-                'Unable to send notification for S3 bucket "%s" to Lambda function "%s".',
-                bucket_name,
-                lambda_function_config,
+                f'Unable to send notification for S3 bucket "{bucket_name}" to Lambda function "{lambda_function_config}".'
             )
 
-    if not filter(lambda x: notif.get(x), NOTIFICATION_DESTINATION_TYPES):
+    if "EventBridge" in notification:
+        s3api_client = aws_stack.connect_to_service("s3")
+        region = (
+            s3api_client.get_bucket_location(Bucket=bucket_name)["LocationConstraint"]
+            or config.DEFAULT_REGION
+        )
+        events_client = aws_stack.connect_to_service("events", region_name=region)
+
+        entry = {
+            "Source": "aws.s3",
+            "Resources": [f"arn:aws:s3:::{bucket_name}"],
+            "Detail": {
+                "version": version_id or "0",
+                "bucket": {"name": bucket_name},
+                "object": {
+                    "key": key,
+                    "size": 4,
+                    "etag": object_data.get("ETag", ""),
+                    "sequencer": "0062E99A88DC407460",
+                },
+                "request-id": "RKREYG1RN2X92YX6",
+                "requester": "074255357339",
+                "source-ip-address": source_ip,
+            },
+        }
+
+        if action == "ObjectCreated":
+            entry["DetailType"] = "Object Created"
+            entry["Detail"]["reason"] = f"{api_method}Object"
+
+        if action == "ObjectRemoved":
+            entry["DetailType"] = "Object Deleted"
+            entry["Detail"]["reason"] = f"{api_method}Object"
+
+        if action == "ObjectTagging":
+            entry["DetailType"] = (
+                "Object Tags Added" if api_method == "Put" else "Object Tags Deleted"
+            )
+
+        entry["Detail"] = json.dumps(entry["Detail"])
+
+        try:
+            events_client.put_events(Entries=[entry])
+        except Exception as e:
+            LOGGER.warning(
+                f'Unable to send notification for S3 bucket "{bucket_name}" to EventBridge', e
+            )
```

Comment:
maybe we could log this using the exception level

```suggestion
            LOGGER.exception(
                'Unable to send notification for S3 bucket "%s" to EventBridge', bucket_name
            )
```